try :

 y = 0/0

except :
    
    print("ERROR")

print('next')